app/src/config/constants/index.js

```diff
@@ -157,4 +157,9 @@ APP_CONSTANTS.mock_base_url = {
   local: "http://localhost:5001/requestly-dev/us-central1/handleMockRequest/api/mockv2", // Change port depending upon you firebase functions emulator port
 };
 
+APP_CONSTANTS.APP_TYPE = {
+  REQUESTLY: "requestly",
+  SESSIONBEAR: "sessionbear",
+};
+
```

Comment:
It would be better to add `sessionbear` [`common/constants.js` under `CONSTANTS.APP_MODES`](https://github.com/requestly/requestly/blob/6db0cbf3eb80ea230a530912e54d514122e44b5d/common/constants.js#L10-L16)